lib/watir/rspec/helper.rb

```diff
@@ -21,10 +21,19 @@ def method_missing(name, *args)
         if browser.respond_to?(name)
           Helper.module_eval %Q[
             def #{name}(*args)
-              browser.send(:#{name}, *args) {yield}
+              if block_given?
```

Comment:
I use a custom layer between rspec & watir/selenium which optionally allows block passing on several location. IMHO the original code will always yield a block even if the original call catched  by the method_missing was not, causing weird behavior in my code. The patch will only pass a block if requested.